test/Main.purs

```diff
@@ -5,18 +5,21 @@ import Prelude
 import Effect (Effect)
 import Effect.Console (log)
 
-import Data.Ratio ((%))
-import Data.Rational (Rational)
+import Data.Ratio (Ratio, (%))
+import Data.BigInt (BigInt, fromInt)
 
-import Test.StrongCheck (Result, quickCheck', (===))
-import Test.StrongCheck.Arbitrary (class Arbitrary)
-import Test.StrongCheck.Gen (Gen, chooseInt, suchThat)
-import Test.StrongCheck.Laws (checkLaws)
-import Test.StrongCheck.Laws.Data as Data
+import Test.QuickCheck (Result, quickCheck', (===))
+import Test.QuickCheck.Arbitrary (class Arbitrary)
+import Test.QuickCheck.Gen (Gen, chooseInt, suchThat)
+import Test.QuickCheck.Laws (checkLaws)
+import Test.QuickCheck.Laws.Data as Data
+import Test.QuickCheck.Laws.Data.Field (checkField) as DataField
 
 import Type.Proxy (Proxy(Proxy))
 
-newtype TestRational = TestRational Rational
+type BiggerRational = Ratio BigInt
```

Comment:
I'd prefer to remove this type alias, I think it's easier to read without it.

Reply:
agreed